rfid_antennas_poses_filelist: ["/home/relief-user0/catkin_ws/src/relief-rfid-antennas-poses-logger/results/pose_reader_001625127C5A_antenna_1.txt",
                               "/home/relief-user0/catkin_ws/src/relief-rfid-antennas-poses-logger/results/pose_reader_001625127C5A_antenna_2.txt",
                               "/home/relief-user0/catkin_ws/src/relief-rfid-antennas-poses-logger/results/pose_reader_001625127C5A_antenna_3.txt",
                               "/home/relief-user0/catkin_ws/src/relief-rfid-antennas-poses-logger/results/pose_reader_001625127C5A_antenna_4.txt"]
